internal/complytime/plugins.go

```diff
@@ -84,11 +84,13 @@ func (p PluginOptions) ToMap(pluginId string, logger hclog.Logger) (map[string]s
 				continue
 			} else {
 				if configOption.Default == nil {
-					if pluginId == "openscap" && configOption.Name == "datastream" {
+					if configOption.Required {
+						return selections, fmt.Errorf("missing required %s default in %s", configOption.Name, configPath)
+					} else {
+						logger.Warn(fmt.Sprintf("Missing %s default in %s, set it to empty string", configOption.Name, configPath))
```

Comment:
```suggestion
						logger.Warn(fmt.Sprintf("Missing default value for %s in %s, it will be set to an empty string", configOption.Name, configPath))
```
nit: When reading, I was wondering if I, as a user, would be expected to set the empty string or if the software would to it to me. : )